pkg/lib/synthesize/supported_resources.go

```diff
@@ -0,0 +1,60 @@
+package synthesize
+
+import "fmt"
+
+const (
+	CSVKind                = "ClusterServiceVersion"
```

Comment:
Same comment as above on this being defined in operator-framework/api instead.

Reply:
Agreed, but I think that can be a follow up